examples/internal/integration/integration_test.go

```diff
@@ -1476,26 +1476,46 @@ func TestResponseBody(t *testing.T) {
 }
 
 func testResponseBody(t *testing.T, port int) {
-	apiURL := fmt.Sprintf("http://localhost:%d/responsebody/foo", port)
-	resp, err := http.Get(apiURL)
-	if err != nil {
-		t.Errorf("http.Get(%q) failed with %v; want success", apiURL, err)
-		return
-	}
-	defer resp.Body.Close()
-	buf, err := ioutil.ReadAll(resp.Body)
-	if err != nil {
-		t.Errorf("ioutil.ReadAll(resp.Body) failed with %v; want success", err)
-		return
-	}
+	tests := []struct {
+		name         string
+		url          string
+		wantStatus   int
+		wantResponse string
+	}{{
+		name:         "unary case",
+		url:          "http://localhost:%d/responsebody/foo",
+		wantStatus:   http.StatusOK,
+		wantResponse: `{"data":"foo"}`,
+	}, {
+		name:         "stream case",
+		url:          "http://localhost:%d/responsebody/stream/foo",
+		wantStatus:   http.StatusOK,
+		wantResponse: `{"data":"foo"}`,
```

Comment:
This is not the response I would expect from a streaming RPC. It should be wrapped in the result envelope:
```json
{"result": {"data":"foo"}, "error":""}
```

Reply:
I'd also like to see this test separated from the unary case and test that we can stream two separate messages.